src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -207,14 +208,21 @@ class OperatorStore extends BaseStore {
   }
 
   /**
-   * Check if operator registrable
+   * Get max validators count
    */
-  async isOperatorRegistrable(validatorsRegisteredCount: number) {
+  async getOperatorValidatorsLimit() {
```

Comment:
Please add return type to the function signature.

Reply:
done